<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1801</width>
    <height>926</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="paintableqlabel" name="cam_knife">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>knife cam</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>550</y>
      <width>451</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="btn_StartCams">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Start cams</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_StopCams">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop cams</string>
     </property>
    </widget>
   </widget>
   <widget class="PlotWidget" name="pg_waterlevel" native="true">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>60</y>
      <width>616</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="sldr_WaterThres">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>60</y>
      <width>22</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbx_WaterThresRange">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>400</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="keyboardTracking">
     <bool>false</bool>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>400</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Range:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbx_pumpOn">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pump ON</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1801</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>paintableqlabel</class>
   <extends>QLabel</extends>
   <header>paintableqlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
